### ISSUE TITLE
Could not load webpack manifest file

### ISSUE THREAD

**Issue by rashedshaon** [rluders/oc-theme-angular#13]
I setup the theme as per documentation but can not run the theme. Showing me the follow error.

 
![image](https://user-images.githubusercontent.com/10374820/87509563-a3463180-c693-11ea-9d04-09839ee7b9a9.png)

What I have to do please help.

**Comment by rluders** [comment]
Did you install the `castiron/webpackassets-plugin` package with the composer directly from your OctoberCMS root directory?

**Comment by rashedshaon** [comment]
> Did you install the `castiron/webpackassets-plugin` package with the composer directly from your OctoberCMS root directory?

Yes I installed it directly from OctoberCMS root directory, what I have to do?

**Comment by rluders** [comment]
Well, that is probably all. That is weird, I'm gonna try to reproduce the issue locally.
